README-agile.md

```diff
@@ -0,0 +1,145 @@
+# AGILE scripts
+
+This repository contains start and stop scripts for various [AGILE](http://agile-iot.eu/) components.
+Currently, AGILE is tested on Raspberry PI 2/3 Model B running Raspbian, and on x86_64 running Ubuntu 16.04.
+Since we use docker containers, it should also run in many other environment.
+
+## Requirements
+
+### On Raspberry PI 2/3 Model B running Raspbian
```

Comment:
I prefer if we keep revisions in a separate branch (like they are now) until they are complete. otherwise master may have incorrect info.